packages/configs/eslint-common-jest.json

```diff
@@ -7,5 +7,9 @@
     ],
     "env": {
         "jest": true
+    },
+    "rules": {
+        "security-node/detect-crlf": "off",
+        "xss/no-mixed-html": "off"
     }
```

Comment:
If we are going to use `security-node/*` and `xss/*` rules, shouldn't we add `security-node` and `xss` plugins in this file?

Reply:
Actually these rules are turned off here in case they are enabled by a different configuration. If the rules are absent (no plugin used) then this does nothing.